Project/backend/.env.example

```diff
@@ -17,3 +17,6 @@ POSTGRES_PORT=5432
 POSTGRES_HOST=amos-db
 
 JANUS_PORT=8182
+
+#Frontend 
+VITE_BACKEND_HOST= 'http://localhost:8000'
```

Comment:
You don't need to add this in the BE `.env.example`. It is already in the FE `.env.development`

Reply:
oh yes thnaks!